src/ParallelAlgorithms/LinearSolver/Actions/BuildMatrix.hpp

```diff
@@ -76,6 +77,18 @@ struct EnableDirectSolve {
       "This can be unfeasible if the linear problem is too big."};
 };
 
+struct SkipResets {
```

Comment:
It would probably good in a future PR to document all the tags in this file. Just a brief dox string so they are discoverable. I'm also fine with you adding a commit at the end of this PR to do it.